README.md

```diff
@@ -1,86 +1,307 @@
-# Tombolo Digital Connector
+<a href="http://www.tombolo.org.uk/products/">
+<p align="center"> <img src="http://www.tombolo.org.uk/wp-content/uploads/2017/09/xDigital-Connector-Icon.png.pagespeed.ic.17pEUiGbW4.png" width="256" height="256"/>
+</p></a>
+<h1 align="center"> Tombolo Digital Connector</h1>
+
 [![wercker status](https://app.wercker.com/status/2279bdc90688501386b12c693be6a186/s/master "wercker status")](https://app.wercker.com/project/byKey/2279bdc90688501386b12c693be6a186)
 
 The [Tombolo Digital Connector](http://www.tombolo.org.uk/products/) is an open source tool that allows data enthusiasts to efficiently connect different data sets into a common format. It enables the **transparent** and **reproducible** combination of data which exists in different domains, different formats and on different spatio-temporal scales. The Tombolo Digital Connector makes it easier to generate models, indexes and insights that rely on the combination of data from different sources.
 
 There are three particularly important parts to the Tombolo Digital Connector: 
 
-- Importers
+- [***Importers***](https://github.com/FutureCitiesCatapult/TomboloDigitalConnector/wiki/Importers)
   - Built-in importers harvest a range of data sources into the centralised data format. Examples include data from ONS, OpenStreetMap, NOMIS, the London Air Quality Network and the London Data Store. **We welcome the creation of additional importers**.
-- Centralised data format
+- [***Centralised data format***](https://github.com/FutureCitiesCatapult/TomboloDigitalConnector/wiki/Local-Datastore)
   - All data imported into the Tombolo Digital Connector adopts the centralised data format. This makes it easier to combine and modify data from different sources.
-- Recipes
+- [***Recipes***](https://github.com/FutureCitiesCatapult/TomboloDigitalConnector/wiki/Recipe-Language)
   - Users generate recipes with a declarative 'recipe language' to combine the data in different ways. This combination can generate new models, indexes and insights. For example, [existing recipes](https://github.com/FutureCitiesCatapult/TomboloDigitalConnector/tree/master/src/main/resources/executions/examples) can generate models of social isolation, calculate the proportion of an area covered by greenspace and even generate an active transport index. **We welcome the creation of additional recipes**.
```

Comment:
@sassalley what do you think?

Reply:
I agree that it can't generate insights. However, I worded it to stress that it `makes it easier to generate insights` not that it generates insights its self. Again - we can certainly rephrase this if it helps.